docs/config_options.md

```diff
@@ -192,6 +192,8 @@ If you define these options you will enable the associated feature, which may in
   * run RGB animations
 * `#define RGBLIGHT_LAYERS`
   * Lets you define [lighting layers](feature_rgblight.md?id=lighting-layers) that can be toggled on or off. Great for showing the current keyboard layer or caps lock state.
+* `#define RGBLIGHT_MAX_LAYERS`
+  * Defaults to 8. Can be expanded up to 32 if more [lighting layers](feature_rgblight.md?id=lighting-layers) are needed.
```

Comment:
```suggestion
  * Defaults to 8. Can be expanded up to 32 if more [lighting layers](feature_rgblight.md?id=lighting-layers) are needed. 
  * Note: Increasing the maximum will increase the firmware size.
```

Reply:
It will also slow down the sync of data of split comms

Reply:
I just pushed a change to add a note about split comms to both doc locations.